DiseasesExpanded/Buildings/Germcatcher.cs

```diff
@@ -45,6 +45,10 @@ public static void InitalizeFlaskDict()
                 SpawnedFlasks.Add(GermIdx.BogInsectsIdx, BogBugsFlask.ID);
             if (DlcManager.IsExpansion1Active() && Settings.Instance.HungerGerms.IncludeDisease)
                 SpawnedFlasks.Add(GermIdx.HungerGermsIdx, HungermsFlask.ID);
+
+            SpawnedFlasks.Add(GermIdx.RoseScentIdx, PollenFlask.ID);
+            SpawnedFlasks.Add(GermIdx.LavenderScentIdx, PollenFlask.ID);
+            SpawnedFlasks.Add(GermIdx.MallowScentIdx, PollenFlask.ID);
```

Comment:
Add bool variable to ModInfo that informs if FragrantFlowers is enabled. Use CheckForRelatedMods to initalize the variable. surround this block of code with if(ModInfo.IsFragrantFlowersEnabled)

Reply:
Fixed.